packages/talker_logger/lib/src/logger_io.dart

```diff
@@ -0,0 +1,3 @@
+import 'dart:io';
+
+void outputLog(String message) => message.split('\n').forEach(print);
```

Comment:
Please use **stdout.writeln** here 
**print** method used in **TalkerFlutter.init()** method from talker_flutter lib

https://github.com/Frezyx/talker/blob/master/packages/talker_flutter/lib/src/extensions/talker_flutter.dart

Reply:
Thank you for your kind guidance. As you suggested, I've changed "print" to "stdout.writeln". Thank you.